documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -3,16 +3,21 @@
 // assembly-kafka-connect.adoc
 
 [id='proc-deploying-kafkaconnector-{context}']
-= Deploying a `KafkaConnector` resource to Kafka Connect
+= Deploying a `KafkaConnector` resource
 
-This procedure describes how to deploy the example `KafkaConnector` to a Kafka Connect cluster.
+You can configure and deploy a `KafkaConnector` resource to a Kafka Connect cluster. 
+This creates a new connector instance with the specified configuration. 
 
-The example YAML will create a `FileStreamSourceConnector` to send each line of the license file to Kafka as a message in a topic named `my-topic`.
+This procedure creates a basic `FileStreamSourceConnector` that sends each line of the Kafka license file to a Kafka topic. 
```

Comment:
I think if this is an example then it's fine to use the license file (we should just say that the license file is just an example file), but if it's really a procedure we should make the file a placeholder and talk about "each line of a file in the container filesystem". We should also be clear that using these connectors in a container isn't useful for any real world use cases, and provided merely as an example of how to configure a connector. Real world use cases would require the preparation of a connect image containing the connectors to be used.

Reply:
I consider this as an example. We provide an example KafkaConnector and this procedure illustrates how to use it.

I've added some new content about production use cases, etc.